src/main/java/io/github/thebusybiscuit/slimefun4/utils/PatternUtils.java

```diff
@@ -26,6 +26,5 @@ private PatternUtils() {}
     public static final Pattern ALPHANUMERIC = Pattern.compile("[A-Fa-f0-9]+");
     public static final Pattern NUMERIC = Pattern.compile("[0-9]+");
 
-    public static final Pattern NUMBER_SEPERATOR = Pattern.compile(",|\\.");
-
+    public static final Pattern NUMBER_SEPARATOR = Pattern.compile("[,.]");
```

Comment:
Same here, wouldn't . be treated as a wildcard?

Reply:
See comment above